### PR TITLE
Don't assume the use of static_web.yml

### PR DESCRIPTION
**Pull request by zmb3**
The `web_ip` variable is typically only defined when using `static_web.yml`, which assumes a single web instance.

Most deployments will have >1 web instances, so this file and variable will not be defined.  Use `external_url` instead of `web_ip` to allow for an arbitrary number of web instances behind the same load balancer.

### CODE REVIEW

File: cluster/operations/uaa.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     consumes: {database: {from: db}}
     properties:
       uaa:
-        url: &uaa-url "https://((web_ip)):8443"
+        url: &uaa-url "https://((external_url)):8443"
```
Comment (vito) [inline cluster/operations/uaa.yml]: Would this end up as `https://https://`?

**Comment by zmb3** [comment]
Great catch, yes. Will push a fix tomorrow.

**Comment by zmb3** [comment]
@vito - Removed the extra `https://`

**Comment by vito** [comment]
Thanks!

Edit: oh, it needs a rebase now.

**Comment by zmb3** [comment]
All set ✅

**Comment by zmb3** [comment]
Friendly ping @vito :-)

**Comment by vito** [comment]
Woops! Merging, thanks!

**Pull request merged**